src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -628,10 +892,11 @@ public static partial class Kernel32
         /// </returns>
         /// <remarks>Minimum OS: Windows Vista / Windows Server 2008.</remarks>
         [DllImport(api_ms_win_core_psapi_l1_1_0, SetLastError = true, CharSet = CharSet.Unicode)]
-        public static extern bool QueryFullProcessImageName(
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern unsafe bool QueryFullProcessImageName(
             SafeObjectHandle hProcess,
             QueryFullProcessImageNameFlags dwFlags,
-            StringBuilder lpExeName,
+            [Friendly(FriendlyFlags.Out | FriendlyFlags.Array)] char* lpExeName,
```

Comment:
Do we generate the "StringBuilder" overload ? If not I think we should (Also semver compat)

Always generating StringBuilder for out + char* and string for in + char* seem like good defaults no ?